packages/admin/blocks-admin/src/blocks/helpers/createCompositeBlockSelectField.tsx

```diff
@@ -0,0 +1,31 @@
+import { SelectField, SelectFieldProps } from "@comet/admin";
+import { MenuItem } from "@mui/material";
+import * as React from "react";
+import { FormProps } from "react-final-form";
+
+import { BlocksFinalForm } from "../../form/BlocksFinalForm";
+import { createCompositeSetting } from "./composeBlocks/createCompositeSetting";
+
+interface Options<T extends string | number> {
+    defaultValue: T;
+    options: Array<{ value: T; label: React.ReactNode }>;
+    formProps?: Partial<FormProps<Record<"value", T>, Record<"value", T>>>;
```

Comment:
Same as in https://github.com/vivid-planet/comet/pull/1699#discussion_r1510918125